README.md

```diff
@@ -25,7 +25,7 @@ Install on your own machine:
 
 - [One-line Docker Preview](http://docs.mattermost.com/install/docker-local-machine.html#one-line-docker-install) 
 - [Developer Machine Setup](https://docs.mattermost.com/developer/dev-setup.html)
```

Comment:
This URL can be changed to: https://developers.mattermost.com/contribute/server/developer-setup/